### PR TITLE
chore: updated home page title

### PR DESCRIPTION
**Pull request by ToddPeltzer**

### CODE REVIEW

**Comment by tdarrow7** [comment]
I like

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export default function Home() {
   return (
     <main>
-      <h1>DarrowCraft</h1>
+      <h1>DarrowCraft's Coffee</h1>
```
Comment (tdarrow7) [inline src/app/page.tsx]: Interesting choice

**Review by tdarrow7 — COMMENTED** [review]
I left a comment on the file

File: src/app/page.tsx (hunk 1 above)
Comment (ToddPeltzer) [reply src/app/page.tsx]: Sorry, I'll add detail

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export default function Home() {
   return (
     <main>
-      <h1>DarrowCraft's Coffee</h1>
+      <h1>DarrowCraft's Premium Coffee</h1>
```
Comment (tdarrow7) [inline src/app/page.tsx:4]: nicely done!

**Review by tdarrow7 — APPROVED** [review]
I approve

**Pull request merged**